contracts-periphery/script/DeployBatchSend.s.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import {Script} from "forge-std/Script.sol";
+import {console2} from "forge-std/console2.sol";
+import {UmbraBatchSend} from "src/UmbraBatchSend.sol";
+import {IUmbra} from "src/interface/IUmbra.sol";
+
+/// @dev Deploys the UmbraBatchSend contract to all supported networks networks, skipping
+/// networks that already have the contract
+/// @dev Set EXPECTED_NONCE constant with the nonce of the deployer address,
+/// which you can get using `cast nonce --rpc-url <url> <address>`
+/// @dev Run the script with `forge script Deploy --private-key <privateKey>`
+/// and add `--broadcast` to broadcast the tx. We *MUST* deploy using the `--private-key`
+/// flag to ensure `msg.sender` is the deployer address, which is used to check the
+/// nonce and compute the expected contract address.
+/// @dev Specify additional network aliases in [rpc_endpoints] in `forge.toml`
+/// before adding them to the list of networks below, and add corresponding environment
+/// variables to the `.env` and `.env.template` files.
+contract Deploy is Script {
+  uint256 constant EXPECTED_NONCE = 0; // Edit this with the nonce of the deployer address
```

Comment:
let's make it obvious there's an outstanding TODO here
```suggestion
  uint256 constant EXPECTED_NONCE = 0; // TODO Edit this with the nonce of the deployer address
```

Reply:
Added https://github.com/ScopeLift/umbra-protocol/pull/504/commits/b62b1a08e70fd154776ac2507ef855007bb00668